commands/bmc.vegman

```diff
@@ -190,6 +194,71 @@ function cmd_ifconfig {
   netconfig "$@"
 }
 
+# @doc cmd_syslog
+# Remote logging settings
+function cmd_syslog {
+  subcommand "$@"
+}
+
+# @sudo cmd_syslog_show admin,operator,user
+# @doc cmd_syslog_show
+# Show actual remote syslog server
+function cmd_syslog_show {
+    local addr="$(busctl get-property ${SYSLOG_CONFIG_SERVICE} \
+                                      ${SYSLOG_CONFIG_PATH} \
+                                      ${NETWORK_CLIENT_IFACE} Address \
+                  2>/dev/null)"
+    addr=${addr#*\"}
+    addr=${addr%\"*}
+
+    local port="$(busctl get-property ${SYSLOG_CONFIG_SERVICE} \
+                                      ${SYSLOG_CONFIG_PATH} \
+                                      ${NETWORK_CLIENT_IFACE} Port \
+                  2>/dev/null)"
+    port=${port#* }
+
+    echo -n "Remote syslog server: "
+    if [[ -z "${addr}" ]] || [[ "${port}" == "0" ]]; then
+        echo "(none)"
+    else
+        echo -n "${addr}"
+        if [[ ! -z "${port}" ]]; then
+            echo -n ":${port}"
+        fi
+        echo
+    fi
+}
+
+# @sudo cmd_syslog_set admin
+# @doc cmd_syslog_set
+# Configure remote syslog server
+#  address[:port] - to change settings
+#  empty - to reset settings
```

Comment:
Not needed anymore.

Reply:
It still works as far as I can see

Reply:
I mean, lets have the one only official way to do it.

Reply:
I think that `bmc syslog set` is shorter than `bmc syslog reset` ;)
And both ways works now.

Reply:
So, now you have to describe both ways in the Users' documentation ;-)